modules/serverless-sinkbinding-kn.adoc

```diff
@@ -14,11 +14,14 @@ This guide describes the steps required to create, manage, and delete a SinkBind
 
 .Procedure
 
-. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log:
+. To check that SinkBinding is set up correctly, create a Service to use as an event sink:
 +
 ----
 $ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
 ----
++
+The command shown creates a Service named `event-display` that uses a sample container image.
+// TODO: Add docs about creating Services using container images.
```

Comment:
That's a weird TODO, as this done already. Here: https://docs.openshift.com/container-platform/4.5/serverless/knative_serving/serving-kn-commands.html#basic-workflow_serving-kn-commands

Reply:
We have a lot of improvements to make around services, this is just to remind me when we do to update some things here.